### PR TITLE
Community: Adding PredictionGuardEmbeddings Class for embeddings generation with Prediction Guard

### PR DESCRIPTION
**Pull request by jmansdorfer**
- *[x] **PR title**: "community: add Prediction Guard embeddings endpoint"

- *[x] **PR message**:
    - **Description:** This PR introduces the `PredictionGuardEmbeddings` class. This enables users to generate embeddings through the Prediction Guard ecosystem.
    - **Issue:** None
    - **Dependencies:** None
    - **Twitter handle:** [@predictionguard](https://x.com/predictionguard)

- *[x] **Add tests and docs**: Notebooks and integration tests are added and working for the new integration feature.

- *[x] **Lint and test**: Linting tests are all passing.

### CODE REVIEW

**Comment by jmansdorfer** [comment]
@efriis

**Comment by efriis** [comment]
Hey! This adds a net-new community integration or feature, which has been replaced by dedicated integration packages. I'll close this PR, and would recommend reopening with just docs updates, as well as registering your package in `libs/packages.yml`! We'll be able to review simple PRs that only modify these two things much faster :)

Here's the guide, and if you have questions, feel free to leave them in the comments on those pages so others can see them! https://python.langchain.com/docs/contributing/how_to/integrations/

File: docs/docs/integrations/providers/predictionguard.mdx
Comment (efriis) [inline docs/docs/integrations/providers/predictionguard.mdx:1]: would also recommend including the llm integration in that other PR, and we're happy to help with deprecation and pointing users to your new package!

Comment (jmansdorfer) [reply docs/docs/integrations/providers/predictionguard.mdx:1]: Will do! Thanks for the update, I saw the partner package update in the docs but wasn't sure whether the to switch to that was finished.

**Pull request closed without merging**